pluginconfig/commit.go

```diff
@@ -157,6 +157,12 @@ type CommitOffchainConfig struct {
 	// NOTE: this can only be used if RMNEnabled == false.
 	MaxMerkleRootsPerReport uint64 `json:"maxRootsPerReport"`
 
+	// MaxPricesPerReport is the maximum number of prices to include in a single report. When this option is set,
+	// the commit plugin will split the prices across multiple reports if there are more than this number of prices.
+	// It will also split prices and merkle roots into separate reports.
+	// NOTE: this can only be used if RMNEnabled == false.
+	MaxPricesPerReport uint64 `json:"maxPricesPerReport"`
+
 	// MultipleReportsEnabled is a flag to enable/disable multiple reports per round.
 	// This is typically set to true on chains that use 'MaxMerkleRootsPerReport'
 	// in order to avoid delays when there are reports from multiple sources.
```

Comment:
Is there no cap to the max # of reports generated in a round? e.g imagine 100 source chains, so we have like 100 merkle roots and 75 price reports, would we generate 100 reports for that round?

Reply:
Yes, in that case up to 175 reports could be produced in 1 round.

Reply:
Up to 175, thats if we don't combine MR's with price updates. What flag allows / disallows that?

Reply:
At least my understanding was if MaxMerkleRoots == 1 and MaxPriceReports == 1 then you could have a report w/ 1 MR and 1 price report

Reply:
No. If you set `MaxPriceReports > 0` then merkle root reports will be separated from the price reports. I'll update the MaxPriceReports comment to clarify the behavior.

Reply:
I see, yeah this info would be good to have.